### PR TITLE
Aboh

### PR DESCRIPTION
**Pull request by ndiayea**

### CODE REVIEW

File: escuela-api/src/main/java/com/escuela/api/controllers/RegistrationController.java [hunk 1]
```diff
@@ -24,5 +25,13 @@ public Registration newUser(@RequestBody Registration newUser){
         return registrationRepository.save(newUser);
     }
 
+    @GetMapping("/finduser")
+    public List<Registration> findRegisteredUsers(HttpServletRequest httpServletRequest){
```
Comment (escuelatech) [inline escuela-api/src/main/java/com/escuela/api/controllers/RegistrationController.java]: Remove the blank spaces

File: escuela-api/src/main/java/com/escuela/api/jpa/repositories/RegistrationRepository.java [hunk 2]
```diff
@@ -3,14 +3,21 @@
...
 
 @Repository
 public interface RegistrationRepository extends JpaRepository<Registration,Integer> {
 
-    @Query("SELECT R FROM Registration R WHERE R.email=?1")
-    List<Registration> findUserByEmailID(String email);
+   @Query("SELECT R FROM Registration R WHERE R.email=?1")
+   List<Registration> findUserByEmailID(String email);
+
+  // @Query()
```
Comment (escuelatech) [inline escuela-api/src/main/java/com/escuela/api/jpa/repositories/RegistrationRepository.java]: Remove commented code

Comment (ndiayea) [reply escuela-api/src/main/java/com/escuela/api/jpa/repositories/RegistrationRepository.java]: @escuelatech Removed Commented code.

File: escuela-api/src/main/java/com/escuela/api/controllers/RegistrationController.java (hunk 1 above)
Comment (ndiayea) [reply escuela-api/src/main/java/com/escuela/api/controllers/RegistrationController.java]: @escuelatech  removed blank spaces.

**Review by escuelatech — APPROVED** [review]

**Pull request merged**